src/main/java/seedu/task/model/task/ReadOnlyTask.java

```diff
@@ -11,6 +11,8 @@
     Description getDescription();
     DueDate getDueDate();
     Duration getDuration();
+    String getDurationStart();
+    String getDurationEnd();
```

Comment:
Do you need these? You should be able to get this information from `getDuration`